src/main/java/cart/application/OrderService.java

```diff
@@ -0,0 +1,65 @@
+package cart.application;
+
+import cart.domain.CartItem;
+import cart.domain.Member;
+import cart.domain.Order;
+import cart.domain.OrderItem;
+import cart.exception.CartItemException;
+import cart.exception.OrderException;
+import cart.exception.OrderException.EmptyItemInput;
+import cart.repository.CartItemRepository;
+import cart.repository.OrderRepository;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class OrderService {
+
+    private final OrderRepository orderRepository;
+
+    private final CartItemRepository cartItemRepository;
+
+    public OrderService(final OrderRepository orderRepository, final CartItemRepository cartItemRepository) {
+        this.orderRepository = orderRepository;
+        this.cartItemRepository = cartItemRepository;
+    }
+
+    public Order createDraftOrder(final Member member, final List<Long> cartItemIds) {
+        if (cartItemIds.isEmpty()) {
+            throw new EmptyItemInput();
```

Comment:
```suggestion
            throw new EmptyItemInputException();
```
네이밍에 Exception이 없어서 예외인지 한 번 더 확인했네요 ㅎㅎㅎ😅 명시해주지 않은 특별한 이유가 있을까요??

Reply:
앗 이건 OrderException.EmptyItemInput 예외인데, 앞 부분을 실수로 static import하여 명시적으로 Exception이 드러나지 않았던 것 같습니다. OrderException.EmptyItemInput()으로 변경하였습니다!

Reply:
static import는 제가 놓쳤네요 😅
결과적으로 지금과 같이 OrderException과 같이 static import 등으로 잘못 사용하는 경우가 있을 수 있는 경우에는 다소 중복처럼 보이더라도 Exception을 명시적으로 선언해주면 어떨까요??

`명시적 표현 vs 간결한 표현`이 고민되는 경우, 명시적으로 적어주는게 휴먼에러나 오용을 방지하는 등 장점이 많지 않을까요??

Reply:
맞습니다. 이번 경우처럼 static import로 예외인지 모르게 하는 실수였어서 명시적 표현이 좋더라구요! 전부 변경하였습니다. 마찬가지로 제 코드를 보시면 내부 메서드나 필드는 전부 this를 표기하도록 되어있는데요, 이또한 명시적으로 표현하기 위한 방법이였습니다~!